core/embed/rust/src/ui/model_tt/layout.rs

```diff
@@ -463,6 +463,7 @@ struct ConfirmBlobParams {
     info_button: bool,
     hold: bool,
     chunkify: bool,
+    text_mono: bool,
```

Comment:
`ConfirmBlob` should by definition be monospace.

why are you using `ConfirmBlob` if you want non-mono text?

Reply:
`ConfirmBlobParams` is used also for `confirm_value`, which is used there. 
I had a problem with `confirm_text`, I'll re-check what was it.

Reply:
As discussed on Slack. 
Other options do not allow using :information_source:  button with Stake/Claim address. 
We will keep it for now but a better solution might to provide info items for `confirm_action`